demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php

```diff
@@ -463,6 +525,12 @@ static function (Statement $statement, array $newValue): array {
                 });
         }
 
+        if ($this->resourceTypeStore->getStatementVoteResourceType()->isAvailable()) {
```

Comment:
No blocker but I think we should avoid using $this->resourceTypeStore AND $this->getTypes(). We should use one of them.

Reply:
@breierDemos good idea, here the change 9af8632b50d24e645ba5e73b18a7cfbebb9aa0b6